app/src/main/java/com/nononsenseapps/feeder/ui/compose/editfeed/EditFeedScreen.kt

```diff
@@ -353,7 +353,7 @@ fun ColumnScope.LeftContent(
         label = {
             Text(stringResource(id = R.string.url))
         },
-        isError = viewState.isNotValidUrl,
+        isError = !viewState.isNostrUri && viewState.isNotValidUrl,
```

Comment:
Could you please move the `isNostrUri` logic inside `isValidUrl` function instead?

Reply:
Will fix.

Reply:
Fixed.